CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to Forgerock's Javascript SDK
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [Forgerock](https://github.com/forgerock) on GitHub.
+
+### Git Flow
+
+Please fork the repository to your own github account. From there, you can add the forgerock repository as an upstream.
```

Comment:
1. Fork the repo to your own github account. 
2. Add the forgerock repository as an upstream.